packages/cms-lib/lib/constants.js

```diff
@@ -73,6 +73,15 @@ const OAUTH_SCOPES = [
   },
 ];
 
+const HUBSPOT_API_KEY = 'HUBSPOT_API_KEY';
+const HUBSPOT_CLIENT_ID = 'HUBSPOT_CLIENT_ID';
+const HUBSPOT_CLIENT_SECRET = 'HUBSPOT_CLIENT_SECRET';
+const HUBSPOT_PERSONAL_ACCESS_KEY = 'HUBSPOT_PERSONAL_ACCESS_KEY';
+const HUBSPOT_PORTAL_ID = 'HUBSPOT_PORTAL_ID';
+const HUBSPOT_REFRESH_TOKEN = 'HUBSPOT_REFRESH_TOKEN';
```

Comment:
Instead of separate constants, I think an enum would be better for this and for the enum to have `EnvironmentVariables` in the name.

Reply:
Fixed